server/database/dbBuild.sql

```diff
@@ -5,16 +5,16 @@ DROP TABLE if EXISTS customer_info CASCADE;
 
 CREATE TABLE customer_info(
   id SERIAL PRIMARY key NOT NULL,
-  first_name VARCHAR(100) NOT NULL,
-  last_name VARCHAR(100) NOT NULL,
+  full_name VARCHAR(100) NOT NULL,
   phone NUMERIC NOT NULL,
   email VARCHAR(50) NOT NULL,
+  zip_code NUMERIC NOT NULL,
   reserve_date TIMESTAMP 
```

Comment:
could you make the reserve_date NOT NULL too?

Reply:
Done!